### DISCUSSION TITLE
tweaking combo dendrons

### DISCUSSION THREAD

**Discussion by SethMilliken** [caksoylar/keymap-drawer#31] [General]
Is there a way to adjust where the dendrons attach to their destinations? There are a few places where a dendron bisects the glyphs on keys that it passes through, or overlaps with other dendrons in an undesirable way. If it were possible to nudge the anchor slightly, that would allow me to clean things up a bit.

This one bisects `\` and the option glyph, overlaps with the command combo dendron, _and_ overlaps with the `T` in `RCTRL`:
<img width="299" alt="Screenshot 2023-07-08 at 17 35 45" src="https://github.com/caksoylar/keymap-drawer/assets/282102/7091f794-7a07-4a0b-b57e-dc20e56aacca">

**Comment by caksoylar** [comment]
Sadly there aren't many knobs to tweak where the dendrons go. The only one right now is a config [`arc_scale`](https://github.com/caksoylar/keymap-drawer/blob/main/keymap_drawer/config.py#L43) which affects all combos globally.

Now that I think about it, it might make sense to convert that to a combo spec field instead, so you can tweak individually. Can you experiment with the above setting a bit and let me know if that would help in this scenario?

(If you don't have a local install, you can try the [web app](https://caksoylar.github.io/keymap-drawer) to iterate quickly where you can paste your custom config at the bottom of the page and give the URL to your keymap as the parse URL input at the "Parse ZMK" box.)

**Reply by SethMilliken** [reply]
Awesome. Nudged it to `1.03` and that did the trick. Per-combo might be nice, but this is probably sufficient for me, at least.
<img width="372" alt="Screenshot 2023-07-08 at 23 05 26" src="https://github.com/caksoylar/keymap-drawer/assets/282102/78e7f9a7-ddf2-4caf-b99b-5b7da61da7d7">

**Reply by caksoylar** [reply]
FYI I ended up moving that property to a combo field instead at https://github.com/caksoylar/keymap-drawer/commit/55c75750ba545c0bdbc13f5f67f45de3d6a3e2c9, so you might need to modify your keymap if you use the `main` branch, or after the next release happens (probably in the next week or so?).
